modules/vba_documents/app/mailers/vba_documents/unsuccessful_report_mailer.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module VBADocuments
+  class UnsuccessfulReportMailer < ApplicationMailer
+    RECIPIENTS = %w[
```

Comment:
Do you think we should load these from the settings file instead of hard coding them?

Reply:
What benefit do you think that would provide? -- I'm actually missing a settings flag for only enabling it in prod

Reply:
I put the switch in here: https://github.com/department-of-veterans-affairs/vets-api/pull/3429/commits/4592aef88076b3cb7fa02b3011f0bf80e76f7c15 do you think we still need a setting for the list?

Reply:
Na its probably ok just to leave them hardcoded for now